pkg/testutils/kyma.go

```diff
@@ -80,6 +80,34 @@ func DeleteKymaByForceRemovePurgeFinalizer(ctx context.Context, clnt client.Clie
 	return DeleteCR(ctx, clnt, kyma)
 }
 
+func DeleteKyma(ctx context.Context,
+	clnt client.Client,
+	kyma *v1beta2.Kyma,
+) error {
+	if err := SyncKyma(ctx, clnt, kyma); err != nil {
```

Comment:
Why is this needed? Since if Kyma has already been deleted, then `SyncKyma` will return nil thus will continue and you will get an error when calling `clnt.Delete`.

I suggest to rewrite something like the following:

```
func DeleteKyma(ctx context.Context,
	clnt client.Client,
	kyma *v1beta2.Kyma,
) error {
    err := clnt.Delete(ctx, kyma)
    if client.IgnoreNotFound(err) != nil {
        return fmt.Errorf("updating kyma %w", err)
    }
    return nil
}
```